### ISSUE TITLE
liveness between worker nodes.

### ISSUE THREAD

**Issue by raokrutarth** [raokrutarth/manufacturing-os#32]
See live ness section. heartbeats and making restructure requests to leader.

**Comment by raokrutarth** [comment]
likely to add 10-15 lines given the existing node classes and message classes so far.

**Comment by raokrutarth** [comment]
@chuanqichen new code that starts the demo cluster just merged into master. You should be able to implement the liveness code in `liveness.py` and integrate it with the `SocketBasedNodeProcess` class

**Comment by raokrutarth** [comment]
waiting on PR #32
